kratos/includes/define.h

```diff
@@ -578,6 +578,34 @@ catch(...) { Block KRATOS_THROW_ERROR(std::runtime_error, "Unknown error", MoreI
 #endif
 
 
+#if defined(__clang__)
```

Comment:
could you please add some comments what these macros are doing? Thx

Reply:
done!